src/template.nunjucks

```diff
@@ -0,0 +1,185 @@
+FORMAT: 1A
+{% if api.meta.HOST %}HOST: {{ api.meta.HOST }}{% endif %}
+
+
+
+
+
+{% macro renderExample(example) %}
+{% for copy in example|getCopy %}
+{{ copy.content }}
+{% endfor %}
+
+{% if example|bodyOnly %}
+        {{ example.messageBody.content|indent(8) }}
+{% else %}
+  {% if example.headers and example.headers.exclude('Content-Type').length %}
+    + Headers
+
+      {% for header in example.headers.exclude('Content-Type').content %}
+            {{ header.key.toValue() }}: {{ header.value.toValue() }}{% endfor %}
+
+  {% endif %}
+  {% if example.dataStructure %}
+    {{ example.dataStructure.content|mson|indent(4) }}
+  {% endif %}
+  {% if example.messageBody %}
+    + Body
+
+            {{ example.messageBody.content|pretty|indent(12) }}
+
+  {% endif %}
+  {% if example.messageBodySchema %}
+    + Schema
+
+            {{ example.messageBodySchema.content|pretty|indent(12) }}
+
+  {% endif %}
+{% endif %}
+{% endmacro %}
+
+
+
+
+
+
+{% macro renderParameters(hrefVariables) %}
++ Parameters
+
+  {% for item in hrefVariables.content %}
+    {% set typeAttributes = item.attributes.get('typeAttributes') %}
+    + {{ item.key.toValue() }}{% if item.value.toValue() %}: {{ item.value.toValue() }}{% endif %}{% if typeAttributes %} ({{ typeAttributes.toValue() }}){% endif %}{% if item.value.description %} - {{ item.value.description }}{% endif %}{% endfor %}
+
+{% endmacro %}
+
+
+
+
+
+
+{% macro renderTransaction(transaction) %}
+{% set request = transaction.request %}
+{% set response = transaction.response %}
+{% if request and (request.contentType or request.headers or request.dataStructure or request.messageBody or request.messageBodySchema) %}
++ Request{% if request.contentType %} ({{ request.contentType }}){% endif %}
+{{ renderExample(request) }}
+{% endif %}
+{% if response and response.statusCode %}
++ Response {{ response.statusCode }}{% if response.contentType %} ({{ response.contentType }}){% endif %}
+{{ renderExample(response) }}
+{% endif %}
+{% endmacro %}
+
+
+
+
+
+{% macro renderTransition(transition, href) %}
+{% if transition.title %}
+#### {{ transition.title }} [{{ transition.method }}{% if href %} {{ href }}{% elif transition.computedHref %} {{ transition.computedHref }}{% endif %}]
+{% else %}
+#### {{ transition.method }}{% if href %} {{ href }}{% elif transition.computedHref %} {{ transition.computedHref }}{% endif %}
+{% endif %}
+
+{% for copy in transition|getCopy %}
+{{ copy.content }}
+{% endfor %}
+
+{% if transition.parameters.length %}
+  {{ renderParameters(transition.parameters) }}
+{% endif %}
+
+{% if transition.relation %}
++ Relation: {{ transition.relation }}
+{% endif %}
+
+{% for transaction in transition.transactions.content %}
+  {{ renderTransaction(transaction) }}
+{% endfor %}
+{% endmacro %}
+
+
+
+
+
+{% macro renderResource(resource) %}
+{% if not resource|resourceShorthand %}
+{% set href = null %}
+### {{ resource.title|default('Resource', true) }} [{{ resource.href }}]
```

Comment:
We shouldn't use the default `Resource` here. The following is a valid blueprint.

```apib
# Tesla

# /users

# List [GET]
+ Response 204
```